README.md

```diff
@@ -11,6 +11,8 @@ This package provides:
   for getting the current time/date
   in a specific timezone.
 
+This package supports Pythons 3.10, 3.11 and 3.12.
```

Comment:
😕 This feels like it will very easily fall out of date with the project settings. Could we link to `pyproject.toml` in the contributing guide, rather than here?

Reply:
Sure. I'll update the contributing guide to point there.